tests/virt_autotest/kubevirt_tests_agent.pm

```diff
@@ -84,8 +91,25 @@ sub rke2_agent_setup {
     # Configure rke2-agent service
     my $server_node_token = script_output("ssh root\@$server_ip cat /var/lib/rancher/rke2/server/node-token");
     assert_script_run('mkdir -p /etc/rancher/rke2/');
-    assert_script_run("echo 'server: https://$server_ip:9345' > /etc/rancher/rke2/config.yaml");
-    assert_script_run("echo 'token: $server_node_token' >> /etc/rancher/rke2/config.yaml");
+    assert_script_run("cat > /etc/rancher/rke2/config.yaml <<__END
+server: https://$server_ip:9345
+token: $server_node_token
+kubelet-arg:
+  - cpu-manager-policy=static
+  - kube-reserved=cpu=500m
+  - system-reserved=cpu=500m
+__END
+(exit \$?)");
+
+    # Setup cnv-bridge containernetworking plugin: one of the tests requires
```

Comment:
would you please specify which test needs this in the comment? It will give us some guide when we won't need such setup.

Reply:
From developer's comments:

> Vasily Ulyanov [Mar 18th at 5:43 PM]
> Also, I think we would need to have this https://gitlab.suse.de/virtualization/kubevirt-ci/-/blob/main/cluster/node.sh?ref_type=heads#L112-L120. I am still debugginh, but seems one of the operator tests requires this cni